docs/fans.md

```diff
@@ -0,0 +1,36 @@
+
+## ** How do you do it? ** ![iphone](https://lh6.googleusercontent.com/PUZX7_Li0FoaCsFyqfU1qve0nbcAQO1lBtNWdltyEyWKaB_RhCyYeJsYIBBJguKMf1qo9kYNdx2c1eLZoBlCojaNMeO1ShWsG93PbiFHf2wC5FuBsvKvTqdVWzs0QLmlwO09Q1ik)
+
+Our fans download the "Shenanigan Mobile App" and register for a fan account. Once registered as a fan the user will interact with the easy to use app interface. User's search for their favorite athlete and once found they can join the athlete's fan club. Once joined users can interact with the community and their favorite athletes! The more a user interacts with their athletes of choice and corresponding fan club's the higher the chance of earning rewards and finding rare digital collectibles known as an “NRG Challenge cards.”
+
+*For more indepth of NRG Collectable cards read more at https://shenanigan.substack.com/p/she-teaches-you-about-dathletes*
+
+## * How do they benefit? *![moneybag](https://lh6.googleusercontent.com/HnldgSWanuvnisv_IkMKUvnoEWkk5KIrPl5C52bF0Ft_5qalW7koarsiXUD9DnFV_miiufmxBomIzOgy8SCAPMoobAqtSYhpHIGcapJYvH6ZSysfGn5o224WhixrD9DelON13mPO)
+
+SHE generates a community where fans can directly donate to the success of their favorite athletes while watching their progress and collecting rare collectibles along the way! By donating to one's favorite athletes, fans will be rewarded with first call to limited digital collector cards! Fans can earn "Fan Tokens", by simply holding their favorite athletes trading card.
+
+Fan Tokens are a native in app token which can be used to tip their favorite athletes or level up their digital collector cards increasing their value. Fan tokens can also be locked up (staked) to earn APY! 
+
+SHE also creates a market of challenges and predictions to gamify live broadcasts where spectators (fans) can be incentivized in various ways:
+Once an athlete has created a challenge, fans will have the opportunity to wager their fan tokens on the success or failure of that specific challenge. If the fan is correct in their prediction they are awarded additional fan tokens!
+
+Comparable to baseball cards, NRG cards will track unique stats and attributes that belong to the athlete. These stats will help you follow the progress and changes in you athlete. As the athletes complete more challenges, we'll update your NRG card to reflect those stats. Be more than just a follower, feel like you're a part of the game, specifically the athlete's coach, as an owners of an athlete card you will want to see constant improvement in your athlete!
+
+Collect, trade and invest in your favorite athletes their success is your success! 
+
+*For more indepth of NRG Collectable cards read more at https://shenanigan.substack.com/p/she-teaches-you-about-dathletes*
+
+## *To achieve this; SHE has arrived *![sparkling_heart](https://lh4.googleusercontent.com/5WW4MOcBkrUq2AuMGFdId_SeJRijzWKeHjPtzi5t5oYDbZGnv7MD9LrIK-ut0eF039y30WG1PW68EiKT2LKtYSgVgv8s3wdQAawTFuctWT_aSgg3PcisaMPegRb3uofbGb6JUl2w)
```

Comment:
Again, just curious about what the benefit of repeating is. I'm not opposed and I think it works here. Maybe we should put this in the introduction instead?

Reply:
I think a good intro and outro can be reused. The overall message is tweaked to speak to the audience of that article. 

> I'm curious if we should use the same titles for both pages. I don't quite disagree but I want to here your reasoning for this. Will it get stale?
 
By title do you mean the headings such as "What is Shenanigan  or Benefits" ?

Reply:
OK I think I agree.

Yes that's what I meant

Reply:
Personally I don't think it will get stale, The headings or titles Are universal for athletes and fans. The only thing i feel really should change would be the text.